### ISSUE TITLE
Hover over an attribute/variable during debug shows an unscrollable/broken UI containing value of attribute

### ISSUE THREAD

**Issue by nitinsurana** [mauricecruz/chrome-devtools-zerodarkmatrix-theme#108]
http://imgur.com/fsFzIMk

**Comment by mauricecruz** [comment]
I think this is fixed now. Please verify when you get a chance!.

**Comment by briantully** [comment]
First off, I love, love, love this theme! Thank you for sharing it :)

However, I am seeing this issue as well and I am using the latest version from the Chrome Web Store. When debugging javascript in developer tools with this theme, you can not read the values when mousing over a javascript variable property. A tooltip tries to appear, but the contents are not visible or legible. It appears as though the tooltip does not expand to the correct height/width of the dynamic content if the content is just 1 line. This issue does not occur when using the default theme. 

Here's a screenshot: https://www.dropbox.com/s/9ag3lu98af759q1/chrome-devtools-zdt-theme-issue.png?dl=0

**Comment by mauricecruz** [comment]
@briantully you're welcome! Glad to hear you're enjoying it =).

I'll try to create a snippet so I can recreate a 1 line tooltip. If you have any snippets handy, that would be useful.

**Comment by nitinsurana** [comment]
```
<!doctype html>
<html>
<head>
</head>
<body>

<script>
	var myVariable = 'thsi is going to be a long variable';
	var testVar = '';

	for(var i=0;i<100;i++)
	{
	    testVar+=myVariable+'<br>';
	}
	console.log(testVar);

</script>
</body>
</html>
```


Debug the above script & hover over testVar, it's a single long-line but the entire line is not visible. Possibly, both horizontal & vertical scrolls are not showing up.

**Comment by briantully** [comment]
So I guess this issue is still unresolved?

**Comment by mauricecruz** [comment]
please verify with 1.7.3 when you get a chance.

**Comment by mauricecruz** [comment]
Definitely not fixed yet.

![image](https://cloud.githubusercontent.com/assets/1640686/6854751/7eaf54e0-d3c9-11e4-99d9-32dc3a2c3f3a.png)

**Comment by mauricecruz** [comment]
fixed in 1.7.5

![image](https://cloud.githubusercontent.com/assets/1640686/6991378/a398388a-da5e-11e4-8ae8-6c57ec9d528c.png)
